src/app/app.component.ts

```diff
@@ -38,6 +38,8 @@ export class AppComponent implements OnInit {
   user: PAUser;
   readonly pathConstants = globals.pathConstants;
   loading = true;
+  readonly NOT_FOUND = 404;
```

Comment:
We could add these kind of constants into a dedicated constants file. Hence, we don't have to add it for each service (I guess it is used not only once?).